.drone.yml

```diff
@@ -0,0 +1,184 @@
+---
+kind: pipeline
+type: docker
+name: test
+steps:
+- name: install
+  image: danlynn/ember-cli:3.24.0
+  commands:
+  - npm ci
+- name: lint:js
+  image: danlynn/ember-cli:3.24.0
+  commands:
+  - npm run lint:js
+- name: lint:hbs
+  image: danlynn/ember-cli:3.24.0
+  commands:
+  - npm run lint:hbs
+- name: test
+  image: danlynn/ember-cli:3.24.0
+  commands:
+  - npm run test:ember
+trigger:
+  event:
+    - pull_request
```

Comment:
Ideally this would also run when commits are pushed to main / master directly, but that doesn't seem to be possible without duplicating the pipeline?

https://docs.drone.io/pipeline/triggers/

> You can also combine multiple triggers. Note that all triggers must evaluate to true when combining multiple triggers.